src/platform/site-wide/announcements/config/index.js

```diff
@@ -28,6 +29,12 @@ const config = {
       paths: /^\/$/,
       component: WelcomeToNewVAModal,
     },
+    {
+      name: 'pittsburgh-vamc',
+      paths: /^\/pittsburgh-health-care\/(.)*$/,
```

Comment:
We should probably get a test pattern together for testing these regexes against website paths.

Reply:
Yeah. Should be easy enough to put the regex in a util file and we could test it there